cmd/argocd/commands/app.go

```diff
@@ -641,16 +682,9 @@ func NewApplicationUnsetCommand(clientOpts *argocdclient.ClientOptions) *cobra.C
 	appOpts := cmdutil.AppOptions{}
 	opts := unsetOpts{}
 	var command = &cobra.Command{
-		Use:   "unset APPNAME parameters",
-		Short: "Unset application parameters",
-		Example: `  # Unset kustomize override kustomize image
-  argocd app unset my-app --kustomize-image=alpine
-
-  # Unset kustomize override prefix
-  argocd app unset my-app --namesuffix
-
-  # Unset parameter override
-  argocd app unset my-app -p COMPONENT=PARAM`,
+		Use:     "unset APPNAME parameters",
+		Short:   "Unset application parameters",
+		Example: fmt.Sprintf(appUnsetExample, cliName),
```

Comment:
What do you think about embedding the var here? I like the idea of keeping the docs code close to the code code.

Reply:
@crenshaw-dev Thanks for your guidance
Variables are introduced here because if we edit directly here, the formatting and indentation will not look good, and the code will not be readable if example is very long. If we define example in var, the printed example will have the same format as the code, with no indentation. In the code I read, such as Kubectl, almost all examples are defined by var，So I made the change in the code. If I understand something wrong, please help me correct it，thanks

Reply:
@Fish-pro I think I follow... what if we defined the vars immediately before the `var command` where they're used? That would keep the docs close to the code (so people don't forget to update docs when they change behavior) and also would avoid having a massive blob of text in the middle of a variable initialization.

Reply:
@crenshaw-dev Thanks for your guidance. Now unify example into code and adjust the indentation of `argocd app create -h` example. Is it OK?